### ISSUE TITLE
Error in grid lines when xlog=TRUE

### ISSUE THREAD

**Issue by JensonChou** [gforge/forestplot#26]
Dear Max,
I have met an isuue when using xlog=TRUE. I want to draw a grid line of 0.5. But the grid line is actullay positioned in e^0.5.
```
row_names <- list(list("test = 1", expression(test >= 2)))
test_data <- data.frame(coef=c(0.3, 5.24),
                        low=c(0.1, 3.78),
                        high=c(1.1, 8.55))
forestplot(row_names,
           test_data$coef,
           test_data$low,
           test_data$high,
           zero = 1,
           grid=structure(c(0.5), gp=gpar(color="steelblue", lty=2)),
           cex = 2,
           lineheight = "auto",
           xlog=TRUE,
           xticks=c(0.1,1,10))
```
![Rplot01](https://user-images.githubusercontent.com/50313384/57208951-f2f98980-7008-11e9-9ad8-fee59ae15e1b.jpeg)

I also tried to enter -0.69, which equals to log(0.5,e). But it report an error.

```
> row_names <- list(list("test = 1", expression(test >= 2)))
> test_data <- data.frame(coef=c(0.3, 5.24),
+                         low=c(0.1, 3.78),
+                         high=c(1.1, 8.55))
> forestplot(row_names,
+            test_data$coef,
+            test_data$low,
+            test_data$high,
+            zero = 1,
+            grid=structure(c(-0.5), gp=gpar(color="steelblue", lty=2)),
+            cex = 2,
+            lineheight = "auto",
+            xlog=TRUE,
+            xticks=c(0.1,1,10))
Error in forestplot.default(row_names, test_data$coef, test_data$low,  : 
  All argument values (mean, lower, upper, zero, grid and clip) should be provided as exponentials when using the log scale. This is an intentional break with the original forestplot function in order to simplify other arguments such as ticks, clips, and more.
```

Actually, this issue is exactly the same as #14 and #10 which had been closed.
In #14, the author solved the issue through updating packages from github.  I have tried to update my packages to version 1.7.2, but it doesn't work. In #10, the author didn't state his solution.

I'm really looking forward for your advise.

**Comment by gforge** [comment]
Thanks for the excellent report. Should be easy to fix.

**Comment by gforge** [comment]
Sorry, seems that I forgot to submit the package to CRAN :disappointed: . I've fixed some minor issues and now submitted 1.8 version. If you inpatient you can install the package directly from GitHub using devtools.
